documentation/src/main/asciidoc/topics/ref_server_realm_aggregate.adoc

```diff
@@ -0,0 +1,41 @@
+[id='aggregate-security-realms_{context}']
+= Aggregate security realms
+
+Aggregate realms combine multiple realms: the first one for the authentication steps and the others for loading the identity for the authorization steps.
+For example, this can be used to authenticate users via a client certificate, and retrieve identity from a properties or LDAP realm.
+
+[discrete]
+== Aggregate realm configuration
+
+.XML
+[source,xml,options="nowrap",subs=attributes+,role="primary"]
+----
+include::xml/server_aggregate_realm.xml[]
+----
+
+.JSON
+[source,json,options="nowrap",subs=attributes+,role="secondary"]
+----
+include::json/server_aggregate_realm.json[]
+----
+
+.YAML
+[source,yaml,options="nowrap",subs=attributes+,role="secondary"]
+----
+include::yaml/server_aggregate_realm.yaml[]
+----
+
+== Name rewriters
```

Comment:
We should document these in the context of the cache-container Authorization element.

Reply:
Done